raiden/tests/unit/test_tokennetwork.py

```diff
@@ -698,511 +695,3 @@ def test_routing_updates(token_network_state, our_address, channel_properties):
     assert new_channel_identifier not in graph_state.channel_identifier_to_participants
     assert len(graph_state.channel_identifier_to_participants) == 0
     assert len(graph_state.network.edges()) == 0
-
-
-def test_routing_issue2663(chain_state, token_network_state, one_to_n_address, our_address):
```

Comment:
Could the issues these tests checked against also appear in the PFS?

Reply:
The expected behavior of the PFS is different in these cases. Both scores of routes and mediation fees are calculated differently. But there are test for roughly comparable cases in the services repo.